coinstacks/ethereum/api/src/index.ts

```diff
@@ -24,7 +26,15 @@ app.get('/health', async (_, res) => {
   })
 })
 
-app.use('/docs', swaggerUi.serve, async (_req: Request, res: Response) => {
+const options = {
+  customCss: '.swagger-ui .topbar { display: none }',
+  customSiteTitle: 'ShapeShift API Docs',
+  customfavIcon: '/favi-blue.png',
+}
+
+app.use(express.static(join(__dirname, '../../../../../../common/api/public')))
+//app.use('/coinstacks/common/api/public/favi-blue.png', express.static('./favi-blue.png'))
+app.use('/docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument, options), async (_req: Request, res: Response) => {
   return res.send(swaggerUi.generateHTML(await import('./swagger.json')))
 })
```

Comment:
Seems like there was a mix of old and new style (setup vs generateHTML)
```suggestion
app.use('/docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument, options))
```

Reply:
I never added that, that was there originally not sure if serving it up for a different function.

Reply:
My suggestion should be all that is needed when using `swaggerUi.setup()`